packages/satcheljs-promise/lib/index.ts

```diff
@@ -0,0 +1,4 @@
+export { promiseMiddleware } from './promiseMiddleware';
+
+import { install } from './install';
+install();
```

Comment:
Not a fan of this pattern - seems like a natural place to call install() would be somewhere in the middleware - like the first time we call the promiseMiddleware.